demo/ChileCompanyCustomerPortal/ChileCompanyMyDashboard.bbj

```diff
@@ -0,0 +1,309 @@
+use ::WebKit/util/ClientUtil.bbj::ClientUtil
+use ::BBjWidget/BBjWidget.bbj::BBjWidget
+use ::WebKit/demo/ChileCompanyCustomerPortal/util/DbUtil.bbj::DbUtil
+use ::WebKit/demo/ChileCompanyCustomerPortal/forms/ChileCompanyItemForm.bbj::ChileCompanyItemForm
+use ::WebKit/model/ListView/ListViewDataModel.bbj::ListViewDataModel
+use ::WebKit/model/ListView/ListHeaderDataModel.bbj::ListHeaderDataModel
+use ::WebKit/model/ListView/ListHeaderDataModel.bbj::ListHeaderMenuItem
+use ::WebKit/model/ListView/Column.bbj::ListColumnModel
+use ::WebKit/model/ListView/Field.bbj::Field
+use ::WebKit/model/ListView/Cell.bbj::Cell
+use ::WebKit/framework/ListOverviewPanel/ListOverviewPanel.bbj::ListOverviewPanel
+use ::WebKit/framework/FormControlPanel/FormControlPanel.bbj::FormControlPanel
+use ::WebKit/widgets/ListView/ListView.bbj::ListView
+
+use java.util.HashMap
+use com.basiscomponents.db.ResultSet
+use com.basiscomponents.db.DataRow
+
+
+class public ChileCompanyMyDashboard extends BBjWidget
+    field private DbUtil dbUtil!
+
+    field private ResultSet RS!
+
+    method public ChileCompanyMyDashboard (BBjWindow wnd!)
+        ClientUtil.addLocalCSS("WebKit/demo/ChileCompanyCustomerPortal/ChileCompanyMyDashboard.css")
+        #super!.create(wnd!,wnd!.getAvailableControlID())
+    methodend
+
+    method public ChileCompanyMyDashboard (BBjWindow wnd!, BBjInt id!)
+        ClientUtil.addLocalCSS("WebKit/demo/ChileCompanyCustomerPortal/ChileCompanyMyDashboard.css")
+        #super!.create(wnd!,id!)
+    methodend
+
+    method public ListViewDataModel initalizeItemListDataModel()
+        declare ListViewDataModel listDataModel!
+        listDataModel! = new ListViewDataModel()
+        
+        listDataModel!.setDatabaseName("ChileCompany")
+        listDataModel!.setTableName("Item")
+        listDataModel!.setListPanelCssClassName("itemListPanel")
+        REM listDataModel!.setActionColumnStatus(0)
+
+        displayName_field! = new Field("DESCRIPTION", "DESCRIPTION")
+        id_field! = new Field("ITEM_NUM", "ID")
+        id_field!.setIsPrimaryColumn(1)
+        cost_field! = new Field("COST")
+        weight_field! = new Field("WEIGHT")
+        price_field! = new Field("PRICE")
+
+        idColumnFields! = new BBjVector()
+        idColumnFields!.add(id_field!)
+        idColumnCell! = new Cell(idColumnFields!)
+        idColumn! = new ListColumnModel("Id", 1)
+        idColumn!.setColumnWidth("15%")
+        idColumn!.setCellStructure(idColumnCell!)
+
+        
+        firstColumnFields! = new BBjVector()
```

Comment:
instead of firstColumn, secondColumn can we use something like productDescription, cost?